### PR TITLE
Advanced editor

### PR DESCRIPTION
**Pull request by cyyyu**
Can change font size per line.
Can change font color per character selection.

### CODE REVIEW

**Comment by dfguo** [comment]
why are some css files deleted?

File: plugins/colorbutton/plugin.js [hunk 1]
```diff
@@ -70,21 +84,6 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					attributes: { role: 'listbox', 'aria-label': '' }
 				},
 
-				onRender: function() {
```
Comment (dfguo) [inline plugins/colorbutton/plugin.js:73]: why it was in onRender before and not now?

File: plugins/colorbutton/plugin.js [hunk 2]
```diff
@@ -188,34 +187,51 @@ CKEDITOR.plugins.add( 'colorbutton', {
 
 				editor.fire( 'saveSnapshot' );
 
-				var bookmarks = selection.createBookmarks();
```
Comment (dfguo) [inline plugins/colorbutton/plugin.js:191]: this line was removed. intentional?

**Review by dfguo — COMMENTED** [review]
reviewed. can you also test it with and without rollout?

File: plugins/colorbutton/plugin.js (hunk 1 above)
Comment (cyyyu) [reply plugins/colorbutton/plugin.js:73]: It is not necessary to add 'onSelectChange' event listener at here. 'onRender' will be fired frequently. So I move the code insided to outside

File: plugins/colorbutton/plugin.js (hunk 2 above)
Comment (cyyyu) [reply plugins/colorbutton/plugin.js:191]: Yep. I think it's useless here.

**Comment by cyyyu** [comment]
The skin we are using on prod now is in [strikingly/ckeditor](https://github.com/strikingly/ckeditor), not here . I just copy them to this project. So css files changed.

**Comment by dfguo** [comment]
reviewed

**Pull request merged**